src/site/includes/debug.drupal.liquid

```diff
@@ -0,0 +1,5 @@
+{% if buildtype == 'localhost' %}
+<script>
+  console.info('Metalsmith template generated from content:', {{ debug }});
+</script>
+{% endif %}
```

Comment:
This is so you can open your browser dev tools to see the data structure you're working with from your console.